gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-bin.zip
```

Comment:
Can you refer this change? We want to maintain Gradle 5.1 compatibility. Switching to 5.6 will not make it backward compatible anymore.

Reply:
See in the PR description. I need a withEnv method on GradleRunner. Do you know alternative way?

Reply:
Got it. Let's go with Gradle 5.2 then. That's the [version that introduced the method](https://docs.gradle.org/current/javadoc/org/gradle/testkit/runner/GradleRunner.html#withEnvironment-java.util.Map-). Can you please also update the [minimum compatible Gradle version in the user guide](https://github.com/bmuschko/gradle-docker-plugin/blob/master/src/docs/asciidoc/00-intro.adoc)?

Reply:
Sure, will do so!

Reply:
@bmuschko done, have a look